api/pkg/cmd/stack.go

```diff
@@ -3,15 +3,20 @@ package cmd
 import (
 	"context"
 	"encoding/json"
+	"fmt"
+	"sync"
 
 	"github.com/chanzuckerberg/happy/api/pkg/dbutil"
+	"github.com/chanzuckerberg/happy/api/pkg/setup"
+	"github.com/chanzuckerberg/happy/shared/config"
 	"github.com/chanzuckerberg/happy/shared/model"
+	"github.com/chanzuckerberg/happy/shared/workspace_repo"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 )
 
 type StackManager interface {
-	GetAppStacks(context.Context, model.AppStackPayload) ([]*model.AppStack, error)
+	GetAppStacks(context.Context, model.AppStackPayload) ([]*model.AppStackResponse, error)
```

Comment:
Do you think we need to start introducing shared models into the shared package?

Reply:
They are in shared package already actually